### PR TITLE
docs: re-organize content for better navigation and discovery

### PR DESCRIPTION
**Pull request by EisenbergEffect**
# Pull Request

## 📖 Description

This PR basically just moves things around and renames a few things in our documentation table of contents.

## 👩‍💻 Reviewer Notes

The best way to get a sense of things is probably to run the website. The main changes were in the sidebars.js file. I didn't physically move any files at this time. I just reworked the ToC organization.

## 📑 Test Plan

I ran the site and it worked ;)

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

## ⏭ Next Steps

A few follow-up tasks:

* Re-organize the actual files. - I think we could pull the non-library docs up to a root docs folder, for example. I think a few other things might need to be moved around.
* Add a new section for "Apps and Experiences" where we can cover DI, routing, state management, etc.
* Add a cheat sheet with quick references for various tasks to the resources section
* Probably add an examples sections with copy/paste code samples
* More issues are tracked on GitHub

### CODE REVIEW

**Review by chrisdholt — APPROVED** [review]

File: sites/website/sidebars.js [hunk 1]
```diff
@@ -7,9 +7,10 @@ module.exports = {
         },
         {
             type: "category",
-            label: "Using FAST Components",
+            label: "Using the Components",
```
Comment (awentzel) [inline sites/website/sidebars.js:10]: Could "the" be removed also?  Doesn't seem necessary.

**Review by awentzel — APPROVED** [review]

File: sites/website/sidebars.js (hunk 1 above)
Comment (EisenbergEffect) [reply sites/website/sidebars.js:10]: Well, it's specifically about using our components not just using any set of components. So, I think it should stay for now.

**Review by brookdozer — APPROVED** [review]

**Review by nicholasrice — APPROVED** [review]

**Comment by nicholasrice** [comment]
I like your test plan.

File: sites/website/src/docs/components/getting-started.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ For simplicity, examples throughout the documentation will assume the library ha
 
 ## Add components
 
-With the components registered, add any component to the HTML. That's it! For a comprehensive list of all elements, see the [Components](/docs/components) section.
+With the components registered, add any component to the HTML. That's it! For a comprehensive list of all elements, see the [Components](/docs/components/accordion) section.
```
Comment (janechu) [inline sites/website/src/docs/components/getting-started.md:98]: This is an interesting problem. Does docusaurus allow for a containing menu item to have it's own page?

**Review by janechu — APPROVED** [review]

File: sites/website/src/docs/components/getting-started.md (hunk 2 above)
Comment (EisenbergEffect) [reply sites/website/src/docs/components/getting-started.md:98]: I don't know actually. I realized the current link was broken so made this quick fix. I'll add a follow-up to see if we can configure a landing page for the group itself in a future update.

**Pull request merged**